interface/.dockerignore

```diff
@@ -0,0 +1,4 @@
+bin
+build
+node_modules
```

Comment:
@kurtwheeler: to address your question more directly, though: I think you're suggesting that we keep the `node_modules` on the host and map that into the Docker container. That would more or less leave things as they are.

I think if we have a Docker image that houses `node_modules` in one of its layers and that becomes a base image that our CI process pulls to do the build then we will only have `npm`-related headaches when we have to `docker build` that "build image". Am I making sense?